.travis.yml

```diff
@@ -24,7 +24,9 @@ env:
 matrix:
  fast_finish: true
  allow_failures:
- - env: EMBER_TRY_SCENARIO=ember-canary
+    - env: EMBER_TRY_SCENARIO=ember-release
```

Comment:
It seems that there's an issue with ember-metal streams in a downstream dependency in thew new releases. Until resolved, or a workaround is found, probably better to lock versions